frontend/packages/shared/src/types/ComponentSpecificConfig.ts

```diff
@@ -7,6 +7,46 @@ import type { ActionButtonAction } from 'app-shared/types/ActionButtonAction';
 import type { GridRow } from 'app-shared/types/GridRow';
 import type { HTMLAutoCompleteValue } from 'app-shared/types/HTMLAutoCompleteValue';
 
+type DataModelBindingsForAddress = {
+  address: string;
+  zipCode: string;
+  postPlace: string;
+  careOf?: string;
+  houseNumber?: string;
+};
+
+type DataModelBindingsForCustom = {
+  [id: string]: string;
+};
+
+type DataModelBindingsForGroup = {
+  group: string;
+};
+
+type DataModelBindingsForList = {
+  [id: string]: string;
+};
+
+type DataModelBindingsLikert = {
+  answer: string;
+  questions: string;
+};
+
+type DataModelBindingsList = {
+  list: string;
+};
+
+type DataModelBindingsOptionsSimple = {
+  simpleBinding: string;
```

Comment:
Can we use the type DataModelBindingsSimple here?

Reply:
We could, but this is a separate type in the layout json schemas. I therefore think it's cleaner to keep it separate, to keep it consistent with the json schema. Ideally we should be able to generate these types automatically from the json schemas at some point in the future.